### ISSUE TITLE
AttributeError: 'str' object has no attribute 'get' when passing the IP with -i argument

### ISSUE THREAD

**Issue by jjsendor** [Yelp/osxcollector#71]
When passing IP address with *-i* argument:
```
$ cat some_malware_incidents.json | python -m osxcollector.output_filters.analyze -M -i 'SOME.IP.ADDRESS.HERE'
```
I've got:
```
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/jsendor/Documents/osxcollector/osxcollector/output_filters/analyze.py", line 493, in <module>
    main()
  File "/Users/jsendor/Documents/osxcollector/osxcollector/output_filters/analyze.py", line 490, in main
    run_filter(output_filter)
  File "osxcollector/output_filters/base_filters/output_filter.py", line 93, in run_filter
    final_blobs = output_filter.end_of_lines()
  File "osxcollector/output_filters/base_filters/chain.py", line 67, in end_of_lines
    return self._on_end_of_lines(self._head_of_chain)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 86, in _on_end_of_lines
    final_lines = self._on_end_of_lines(link._next_link)
  File "osxcollector/output_filters/base_filters/chain.py", line 81, in _on_end_of_lines
    for blob in link.end_of_lines():
  File "osxcollector/output_filters/opendns/related_domains.py", line 77, in end_of_lines
    domains = self._find_related_domains(domains, ips)
  File "osxcollector/output_filters/opendns/related_domains.py", line 104, in _find_related_domains
    related_domains.update(self._rr_history_to_domains(rr_history_info))
  File "osxcollector/output_filters/opendns/related_domains.py", line 122, in _rr_history_to_domains
    for rr_domain in rr_history.get('rrs', []):
AttributeError: 'str' object has no attribute 'get'
```

**Comment by ivanlei** [comment]
Fix is building in Travis CI

**Comment by jjsendor** [comment]
Fixed in #72.
